Update README file (#838)
submariner-io/submariner: README.md

[![Linting](https://github.com/submariner-io/submariner/workflows/Linting/badge.svg)](https://github.com/submariner-io/submariner/actions?query=workflow%3ALinting+branch%3Amaster)
[![Release Images](https://github.com/submariner-io/submariner/workflows/Release%20Images/badge.svg)](https://github.com/submariner-io/submariner/actions?query=workflow%3A%22Release+Images%22)
[![Periodic](https://github.com/submariner-io/submariner/workflows/Periodic/badge.svg)](https://github.com/submariner-io/submariner/actions?query=workflow%3APeriodic+branch%3Amaster)
<!-- markdownlint-enable line-length -->

Submariner is a tool built to connect overlay networks of different Kubernetes clusters. While most testing is performed against Kubernetes
clusters that have enabled Flannel/Calico/Canal/Weave/OpenShiftSDN, Submariner should be compatible with any CNI cluster network
provider, as it utilizes off-the-shelf components to establish encrypted tunnels between each Kubernetes cluster.

Note that Submariner is in the **pre-alpha** stage, and should not be used for production purposes. While we welcome usage and

> experimentation with it, it is quite possible that you could run into bugs.

Reviewer: ```suggestion
experimentation, it is quite possible that you could run into bugs.
```

# Installation

> Submariner supports a number of different deployment models, including an Operator and Helm Charts. The Operator can be deployed directly
> or via the `subctl` CLI helper utility. `subctl` greatly simplifies the deployment of Submariner, and is therefore the recommended

Reviewer: ```suggestion
Submariner supports deployment via an Operator as well as Helm Charts. The Operator can be deployed directly
or via the `subctl` CLI helper utility. `subctl` greatly simplifies the deployment of Submariner, and is therefore the recommended
```
